scripts/docker/debian/stretch/Dockerfile

```diff
@@ -98,3 +98,11 @@ USER ${JENKINS_USERID}
 
 RUN git config --global user.email 'Jenkins <autobuilder@libelektra.org>' \
     && git config --global user.name 'Jenkins'
+
+# Handle Haskell dependencies
+WORKDIR /home/jenkins/elektra-cabal-sandbox
+RUN cabal update && \
+cabal sandbox init && \
+cabal install 'containers >=0.4 && <0.6' 'base >=4.7 && <5' 'directory >=1.1 && <1.4' \
+'process >=1.2 && <1.7' 'filepath >=1.3 && <1.5' 'base -any' 'hspec -any' 'QuickCheck -any' && \
+echo 'export HASKELL_SHARED_SANDBOX=/home/jenkins/elektra-cabal-sandbox' >> /etc/bash.bashrc
```

Comment:
That is not how docker works.
Use 
```
ENV HASKELL_SHARED_SANDBOX=/home/jenkins/elektra-cabal-sandbox
RUN mkdir -p $HASKELL_SHARED_SANDBOX && cabal init....
```

And do not forget to add it in the other containers that do haskell testing as well.